src/Starlight/Body.hs

```diff
@@ -123,13 +123,13 @@ systemAt sys@System{ bodies } t = sys { bodies = bodies' } where
   bodies' = Map.mapWithKey go bodies
   go identifier body@Body{ orbit = Orbit{ orientation } } = StateVectors
     { body
-    , transform = rel !*! translated3 (unP (position actor))
+    , transform = rel !*! translated3 (prj <$> position actor)
     , actor = actor
       & position_.coerced.extended 1 %~ (rel !*)
-      & velocity_.extended 0 %~ (rel !*)
+      & velocity_.coerced.extended 0 %~ (rel !*)
     } where
     actor = actorAt body t
-    rel = maybe (systemTrans sys) transform (parent identifier >>= (bodies' Map.!?))
+    rel = maybe identity transform (parent identifier >>= (bodies' Map.!?))
```

Comment:
We no longer apply the system scale to body positions, and thus obviate the need to invert it all over the place.